easybuild/easyconfigs/q/QTLtools/QTLtools-1.1-intel-2016b.eb

```diff
@@ -0,0 +1,41 @@
+# Easyconfig for QTLtools
+# Author: Caspar van Leeuwen
+# SURFsara, Amsterdam, The Netherlands
+
+easyblock = 'MakeCp'
+
+name = 'QTLtools'
+version = '1.1'
+
+homepage = "https://qtltools.github.io/qtltools/"
+description = """QTLtools is a tool set for molecular QTL discovery and analysis.
+It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) 
+in few easy-to-perform steps."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://qtltools.github.io/qtltools/binaries/']
+sources = ['QTLtools_%(version)s_source.tar.gz']
+checksums = ['a6054b2b35f67d3fe046fd013cf09753955eb2758a5762297b8e05a8d64cf9ed']
+
+dependencies = [
+    ('GSL', '2.3'),
+    ('bzip2', '1.0.6'),
+    ('Boost', '1.61.0'),
+    ('Rmath', '3.3.1'),
+    ('HTSlib', '1.4')
+]
+
+# Overwrite CXX, CXXFLAG, LIB_FLAGS and LIB_FILES build options from the default makefile
+# since the default makefile uses hard-coded compiler & library paths
+buildopts = 'CXX="$CXX -std=c++0x" CXXFLAG="$CXXFLAGS" LIB_FILES="" '
+buildopts += 'LIB_FLAGS="-lz -lgsl -lmkl -lbz2 -lm -lpthread -lRmath -lhts -lboost_iostreams -lboost_program_options"'
+
+files_to_copy = ['bin']
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin']
```

Comment:
Please make this more specific, by checking for specific binaries (no need to check all if there are a lot of them).

In this case, I'd use:

```python
sanity_check_paths = {
    'files': ['bin/QTLtools'],
    'dirs': [],
}
```